awslabs/solution-spark-on-aws
# refactor: awsService to use aws-sdk clients

Issue #, if available:

Description of changes:
Instead of writing our own AWS SDK clients, we can use the clients provided by AWS

**Tests**
* Launch/start/stop/terminate/connect API works as expected
* Create hosting account API works
* Running post deployment script works as expected

Checklist:




By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
